client/packages/invoices/src/Returns/DetailView/ContentArea.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { FC } from 'react';
+import {
+  DataTable,
+  useTranslation,
+  Box,
+  //   MiniTable,
+  //   useIsGrouped,
+  //   InvoiceLineNodeType,
+  //   useRowStyle,
+  //   AppSxProp,
+  NothingHere,
+  useUrlQueryParams,
+} from '@openmsupply-client/common';
+// import { useOutbound } from '../api';
+import { OutboundReturnLine, useOutboundReturnColumns } from './columns';
+// import { useExpansionColumns } from './OutboundLineEdit/columns';
+
+interface ContentAreaProps {
+  onAddItem: () => void;
+  onRowClick?: null | ((rowData: OutboundReturnLine) => void);
+  rows: OutboundReturnLine[];
+}
+
+// const Expand: FC<{
+//   rowData: StockOutLineFragment | StockOutItem;
+// }> = ({ rowData }) => {
+//   const expandoColumns = useExpansionColumns();
+
+//   if ('lines' in rowData && rowData.lines.length > 1) {
+//     return <MiniTable rows={rowData.lines} columns={expandoColumns} />;
+//   } else {
+//     return null;
+//   }
+// };
+
+// const useHighlightPlaceholderRows = (
```

Comment:
I think we can remove the placeholder row stuff - we're returning specific stock, so shouldn't be possible to have unallocated return rows.